ast/misc.go

```diff
@@ -1250,6 +1308,19 @@ const (
 	ObjectTypeTable
 )
 
+// Restore implements Node interface.
+func (n ObjectTypeType) Restore(ctx *RestoreCtx) error {
+	switch n {
+	case ObjectTypeNone:
+		// do nothing
+	case ObjectTypeTable:
+		ctx.WriteKeyWord("TABLE ")
```

Comment:
```suggestion
		ctx.WriteKeyWord("TABLE")
```

This also means line 1389 and 1447 would need to look like this 🤷‍♂️:

```go
if n.ObjectType != ObjectTypeNone {
	if err := n.ObjectType.Restore(ctx); err != nil {
		return errors.Annotate(err, "An error occurred while restore RevokeStmt.ObjectType")
	} 
	ctx.WritePlain(" ")
}
```

Reply:
Yes, this is better. Should I create a new pull request to fix it? Sorry for the trouble.

Reply:
@exialin Yes please. Thanks.